src/app-bridge/use-dashboard-token.ts

```diff
@@ -0,0 +1,49 @@
+import debugPkg from "debug";
+import * as jose from "jose";
+import { useMemo } from "react";
+
+import { useAppBridge } from "./app-bridge-provider";
+
+export interface DashboardTokenPayload extends jose.JWTPayload {
+  app: string;
+}
+
+export interface DashboardTokenProps {
+  isTokenValid: boolean;
+  hasAppToken: boolean;
+  tokenClaims: DashboardTokenPayload | null;
+}
+
+const debug = debugPkg.debug("app-sdk:AppBridge");
```

Comment:
Maybe this will be better when trying to find which function caused debug message (same as in middlewares)?

```suggestion
const debug = debugPkg.debug("app-sdk:AppBridge:useDashboardToken");
```